### PR TITLE
OCP4 STIG: SRG-APP-000343-CTR-000780 is inherently met

### PR DESCRIPTION
**Pull request by jhrozek**
#### Description:

Adds the justification and supporting documentation.
Removes the rules that were previously linked as they were spurious.

#### Rationale:

- OCP4 STIG

#### Review Hints:

- Review what's rendered for this control in the SRG export

### CODE REVIEW

**Comment by jhrozek** [comment]
Rendered artifact: https://github.com/ComplianceAsCode/content/suites/10884754233/artifacts/549361197

**Comment by jhrozek** [comment]
Rendered artifact: https://github.com/ComplianceAsCode/content/suites/10884754233/artifacts/549361197

**Comment by jhrozek** [comment]
@Vincent056 all tests passed, PTAL

File: controls/srg_ctr/SRG-APP-000343-CTR-000780.yml [hunk 1]
```diff
@@ -3,8 +3,23 @@ controls:
...
+    Select a log to inspect by listing them first:
+    > oc adm node-logs --role=master --path=openshift-api/server`
+
+    View a specific API server log by providing the node name and the log name:
+    > oc adm node-logs <node_name> --path=openshift-apiserver/<log_name>`
+
+  artifact_description: |-
+    Supporting evidence is in the following documentation
+
+    https://docs.openshift.com/container-platform/latest/security/audit-log-policy-config.html#audit-log-policy-config
```
Comment (rhmdnd) [inline controls/srg_ctr/SRG-APP-000343-CTR-000780.yml:24]: Links are different from what's in the draft. Is this more accurate?

**Review by rhmdnd — COMMENTED** [review]
Looks good, one question inline.

**Comment by jhrozek** [comment]
Closing. This control is no longer inherently met in the new draft, but covered with several RHCOS controls.

**Pull request closed without merging**